libs/wingsdk/src/cloud/api.ts

```diff
@@ -326,3 +322,51 @@ export interface IApiEndpointHandlerClient {
    */
   handle(request: ApiRequest): Promise<ApiResponse>;
 }
+
+/**
+ * Parse an HTTP method string to an HttpMethod enum
+ * @param method HTTP method string
+ * @returns HttpMethod enum
+ * @throws Error if the method is not supported
+ */
+export function parseHttpMethod(method: string): HttpMethod {
+  switch (method) {
```

Comment:
Wouldn't this work?

```js
HttpMethod[method]
```

Reply:
That would work, the only downside is that if it tries parsing an invalid method, it would produce `undefined`. I feel like there could be some value in bubbling that up as an error